core/record.go

```diff
@@ -158,6 +158,31 @@ func (t *Timetrace) loadAllRecords(date time.Time) ([]*Record, error) {
 	return records, nil
 }
 
+func (t *Timetrace) loadAllRecordsSortedAscending(date time.Time) ([]*Record, error) {
```

Comment:
I I decided to create a new function to return the sorted records that breakTime() uses. Let me know if this doesnt seem like a good way to handle getting sorted records. I'm also open to better naming suggestions for loadAllRecordsSortedAscending.

Reply:
Nice! 👍